source/scpd/Cpp.d

```diff
@@ -194,15 +194,16 @@ public alias SCPCallback = extern(C++) void function();
 
 private extern(C++) set!uint* makeTestSet();
 
-unittest
-{
-    auto set = makeTestSet;
-    assert(!set.empty);
-    uint[] values;
-    foreach (val; *set)
-        values ~= val;
-    assert(values == [1, 2, 3, 4, 5]);
-}
+// TODO : For MIPS64 test
```

Comment:
Where did you read about MIPS64 ?
This is MIPS64: https://en.wikipedia.org/wiki/MIPS_architecture
I suppose it was somewhere in Ocean ? In any case, MIPS64 has *absolutely nothing to do* with your issues, so I have absolutely no idea why it is being mentioned here.

Reply:
The problem with this is the mangling issue.
Annotation has been modified 'MSVC mangling issue'

Reply:
Mangling issue does not tell me much... What fails to mangle properly ? Is it `empty` ? `makeTestSet` ? And why ?